README.md

```diff
@@ -77,8 +77,24 @@ To test the build, run `cargo test`.
   ```
 - **Windows**
   
-  On Windows, both `R` and `Rtools` should be available on `PATH` for tests to succeed. Ensure that `R_HOME` points to `R` home, e.g. `C:\Program Files\R\R-4.1.0`. 
-  For `x64`, append the following to the `PATH` (using `PowerShell` syntax):
+  On Windows, first, ensure that `R_HOME` points to `R` home, e.g. `C:\Program Files\R\R-4.1.0` (on an R session, this should be set by R).
+  
+  Then, also ensure `PATH` is properly configured that the following executables are available:
```

Comment:
Perhaps '*Second, ensure that*' is better because you have used 'first' in the paragraph above, which implies there is a 'second' and potentially 'third' and so on.